include/gsl/multi_span

```diff
@@ -1998,14 +1996,13 @@ private:
 
 template <class Span>
 class contiguous_span_iterator
-    : public std::iterator<std::random_access_iterator_tag, typename Span::value_type>
 {
-    using Base = std::iterator<std::random_access_iterator_tag, typename Span::value_type>;
-
 public:
-    using typename Base::reference;
-    using typename Base::pointer;
-    using typename Base::difference_type;
+    using iterator_category = std::random_access_iterator_tag;
+    using value_type = typename Span::value_type;
+    using difference_type = std::ptrdiff_t;
+    using pointer = typename Span::value_type*;
+    using reference = typename Span::value_type&;
```

Comment:
The ``pointer`` and ``reference`` declarations should refer to ``value_type``, not ``Span::value_type``

Reply:
done